config/env/test.example.env

```diff
@@ -1,6 +1,6 @@
 # Application
-SECRET_KEY_BASE=""
-DEVISE_SECRET_KEY=""
+SECRET_KEY_BASE="7ba87bf80efaf54543aaa13f1a83cc598944c2c56411a6657c380156cdf2ef2950ffbbee5d3ae8ac07c65aaa12465b37ce5bbfc905f2adcc8ff18d481f29838d"
+DEVISE_SECRET_KEY="f87a7bcf0ed12b1bf931d320167022f71aab151dbf68cec13ecac75c1fa1f2a62d8a29890a3bd691dc0a4e86cae661872cbbdb7e4216948e4412f7ee33737330"
```

Comment:
Looks like you accidentally committed this! These values should be stored in `test.env`